src/MarkdownTextInput.web.tsx

```diff
@@ -296,6 +305,30 @@ const MarkdownTextInput = React.forwardRef<TextInput, MarkdownTextInputProps>(
       [handleSelectionChange, updateRefSelectionVariables],
     );
 
+    const handleContentSizeChange = useCallback(() => {
+      if (!divRef.current || !multiline || !onContentSizeChange) {
+        return;
+      }
+
+      const hostNode = (divRef.current.firstChild as HTMLElement) ?? divRef.current;
+      const newHeight = hostNode.offsetHeight;
+      const newWidth = hostNode.offsetWidth;
```

Comment:
Just wondering if there's any reason for this order (`height` before `width`), is this intended?

Reply:
There is no reason behind this, is it like better to put `width` first? Does it make any difference?